Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -53,6 +54,44 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
             });
         },
 
+
+        /**
+         * Returns true if a provided item is a NavigationItemView
+         *
+         * @method _isNavigationViewItem
+         * @private
+         * @param {Object} item
+         * @return true if item is a NavigationItemView, false if item is a NavigationItemParameterView
+         */
+        _isNavigationViewItem: function (item) {
+            return item.get;
```

Comment:
Nitpick: if you want it to return a boolean it should actually be `return !!item.get;`